lib/elixir/src/elixir_erl.erl

```diff
@@ -318,7 +318,23 @@ typespecs_form(Map, TranslatedTypespecs, MacroNames) ->
   Forms1 = types_form(Types, Forms0),
   Forms2 = callspecs_form(spec, Specs, [], MacroNames, Forms1, Map),
   Forms3 = callspecs_form(callback, AllCallbacks, OptionalCallbacks, MacroCallbackNames, Forms2, Map),
-  {Types, AllCallbacks, Forms3}.
+  AllCallbacks2 = typespecs_back_to_callbacks(Forms3),
```

Comment:
Are we doing this because of duplicates in AllCallbacks? If so, maybe it would be better to call `lists:uniq` or `lists:usort` in callbacks?

Reply:
Good call, I'm now stripping spec information from callbacks and usorting them, check out latest commit.